src/test/prevector_tests.cpp

```diff
@@ -26,57 +26,70 @@ class prevector_tester {
     pretype pre_vector_alt;
 
     typedef typename pretype::size_type Size;
+    bool passed = true;
+    uint32_t insecure_rand_Rz_cache;
+    uint32_t insecure_rand_Rw_cache;
 
+
+    template <typename A, typename B>
+        void local_check_equal(A a, B b)
+        {
+            local_check(a == b);
+        }
+    void local_check(bool b) 
+    {
+        passed &= b;
```

Comment:
This should more correctly be written with a logical rather than a bitwise operator
(and a bit more verbosity as there is no `operator &&=`)
`passed = passed && b;`
but we can let it go as it works, and it's a test.
(I'm more upset by the strange indentation on the local_check_equal template function above)